opcua/client/client.py

```diff
@@ -6,6 +6,7 @@
 except ImportError:  # support for python2
     from urlparse import urlparse
 
+from concurrent.futures import _base
```

Comment:
why did you add that import? otherwise it looks ok

Reply:
In line 258 (call to self.create_session()), can sometimes fail and I need to clean-up socket connection if this happens.

Reply:
@oroulet, any additional thoughts? I can remove it but that means the user needs to know to clean-up on failure outside of the class.

Reply:
@oroulet, I have submitted new PR without importing concurrent.futures, kindly review.